File: Swift/F/Fizzbuzz.swift
import Foundation

for num in 1...100 {
	switch(num % 3, num % 5) {
		case(0, 0):
			print("Fizzbuzz")
		case(_, 0):
			print("Buzz")
		case(0, _):
			print("Fizz")
		default:
			print(num)
	}
}

File: Java/F/Fizzbuzz.java
class fizzBuzz {
  public static void main(String[] args) {
    for(counter = 1; counter <= 100; counter++) {
      if (counter % 15 == 0) {
        System.out.println("Fizzbuzz\n");
      } 
      else if (counter % 5 == 0) {
        System.out.println("Buzz\n");
      } 
      else if (counter % 3 == 0) {
        System.out.println("Fizz\n");
      } 
      else {
        System.out.println(counter, "\n");
      }
    }
  }
}
